project/src/main/ui/career/career-map-level-select.gd

```diff
@@ -7,6 +7,7 @@ signal level_button_focused(button_index)
 export (PackedScene) var LevelSelectButtonScene: PackedScene
 
 var _duration_calculator := DurationCalculator.new()
+var _prev_focused_level_button_index := -1
```

Comment:
Comment this field.

Reply:
The field name is self-documenting; a comment like `The previously focused level button index` feels redundant and I can't think of anything to add.